lib/redis/reconnect_with_readonly.rb

```diff
@@ -55,9 +55,9 @@ def call_with_reconnect_with_readonly(command, &block)
       end
     end
 
-    def call_loop_with_reconnect_with_readonly(command, &block)
+    def call_loop_with_reconnect_with_readonly(command, timeout, &block)
```

Comment:
Thanks for reviewing this! That's a good idea; since redis-rb's `call_loop` has a default timeout of 0, the replacement probably should too.

I'll commit an update.